### ISSUE TITLE
read.py doesn't start generating autoregressive samples

### ISSUE THREAD

**Issue by BarnabyssJones** [neonbjb/tortoise-tts#533]
When using the read.py script, it gets stuck on 0% Generating autoregressive samples. This same thing occured when I tried to use the do_tts.py script to read several sentences.  I have successfully gotten the do_tts.py script to work for a single phrase.

**Comment by minuske4** [comment]
I'm having the same thing and not sure what's wrong, did you solve this?

**Comment by BarnabyssJones** [comment]
I hate to say it but I haven't. I'm a total beginner to all of this and don't really have any abilities other than to follow instructions as things stand, so I'm stuck.

**Comment by minuske4** [comment]
Hi so I think I got the issue fixed for me but I'm also new to this and am not sure what exactly did it so I'll just list the things here:

1. installed this version of transformers 
```
pip install transformers==4.26.1
```

2. reinstalled pytorch through this line
```
conda install pytorch torchvision torchaudio
```

I think one of these two is what fixed it but to be honest i'm a little fuzzy on what I did since I remember trying quite a few different solutions and coming across various errors in the process. I have also since unfortunately closed out of the terminal so I can't see what exact commands I used and in what order anymore so I am mostly relying on memory here and web browser history for commands I looked up, sorry about that.
